src/index.css

```diff
@@ -8,5 +8,51 @@
   font-family: 'Work Sans', sans-serif;
 }
 
+/* Global Styles */ 
 
+/* Ensures a parent element with this class name stretches to the height of floated children */
+.clearfix:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+}
+
+.primarySection {
+  padding: 120px 0;
+}
+
+.background2 {
+  background-color: gray;
+}
+
+.background3 {
+  background-color: #3b668e;
+  color: white;
+}
+
+.background4 {
+  background-color: white;
+}
+
+.float-left {
+  float: left;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: -20px;
+  margin-right: -20px;
```

Comment:
The `.col-33` has a 20px margin, but you want the column edges to stretch to the full width of the row/container. So, the negative margins on the parent row.